src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -45,21 +45,24 @@
 from psyclone.psyGen import PSyFactory
 from psyclone.psyir import nodes
 from psyclone import psyGen
-from psyclone.psyir.nodes import OMPDoDirective, OMPParallelDirective, \
-    OMPParallelDoDirective, OMPMasterDirective, OMPTaskloopDirective, \
-    OMPTaskwaitDirective, OMPTargetDirective, OMPLoopDirective, Schedule, \
-    Return, OMPSingleDirective, Loop, Literal, Routine, Assignment, \
-    Reference, OMPDeclareTargetDirective, OMPNowaitClause, \
-    OMPGrainsizeClause, OMPNumTasksClause, OMPNogroupClause, \
-    OMPPrivateClause, OMPDefaultClause, OMPReductionClause, \
-    OMPScheduleClause, OMPTeamsDistributeParallelDoDirective, \
-    OMPFirstprivateClause
-from psyclone.psyir.symbols import DataSymbol, INTEGER_TYPE, SymbolTable, \
-    REAL_SINGLE_TYPE, INTEGER_SINGLE_TYPE, Symbol
+from psyclone.psyir.nodes import (
+    OMPDoDirective, OMPParallelDirective,
+    OMPParallelDoDirective, OMPMasterDirective, OMPTaskloopDirective,
+    OMPTaskwaitDirective, OMPTargetDirective, OMPLoopDirective, Schedule,
+    Return, OMPSingleDirective, Loop, Literal, Routine, Assignment,
+    Reference, OMPDeclareTargetDirective, OMPNowaitClause,
+    OMPGrainsizeClause, OMPNumTasksClause, OMPNogroupClause,
+    OMPPrivateClause, OMPDefaultClause, OMPReductionClause,
+    OMPScheduleClause, OMPTeamsDistributeParallelDoDirective,
+    OMPAtomicDirective, OMPFirstprivateClause, OMPSimdDirective)
+from psyclone.psyir.symbols import (
+    DataSymbol, INTEGER_TYPE, SymbolTable,
+    REAL_SINGLE_TYPE, INTEGER_SINGLE_TYPE, Symbol)
 from psyclone.errors import InternalError, GenerationError
-from psyclone.transformations import Dynamo0p3OMPLoopTrans, OMPParallelTrans, \
-    OMPParallelLoopTrans, DynamoOMPParallelLoopTrans, OMPSingleTrans, \
-    OMPMasterTrans, OMPTaskloopTrans, OMPLoopTrans
+from psyclone.transformations import (
+    Dynamo0p3OMPLoopTrans, OMPParallelTrans,
+    OMPParallelLoopTrans, DynamoOMPParallelLoopTrans, OMPSingleTrans,
+    OMPMasterTrans, OMPTaskloopTrans, OMPLoopTrans)
 from psyclone.tests.utilities import get_invoke
 
 BASE_PATH = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(
```

Comment:
Not yours but would you mind fixing:

    psyir/nodes/omp_directives_test.py:573:0: C0325: Unnecessary parens after '=' keyword (superfluous-parens)